src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -665,6 +643,30 @@ def _get_query(keywords, only_open):
   return query
 
 
-def get(project, config, issue_tracker_client=None):
+def get(project, component_id, issue_tracker_client=None):
```

Comment:
This can't be changed and is part of the contract for an issue tracker to be "registered" for ClusterFuzz. 

see e.g. other implementations that do the same thing https://github.com/google/clusterfuzz/blob/709fbb62ca1e2a809dcc6b197076d97d75263115/src/clusterfuzz/_internal/issue_management/monorail/__init__.py#L307